nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -1065,7 +1067,7 @@ def get_nspikes(units_table: pynwb.misc.Units, unit_id: int):
         return units_table["spike_times_index"].data[index] - units_table["spike_times_index"].data[index - 1]
 
 
-def write_units(
+def add_units(
     sorting: SortingExtractor,
```

Comment:
```suggestion
    sorting: SpikeInterfaceSorting,
```
Suggest defining `SpikeInterfaceSorting = Union[SortingExtractor, BaseSorting]` above